samples/apps/copilot-chat-app/webapp/src/libs/services/BaseService.ts

```diff
@@ -32,7 +32,11 @@ export class BaseService {
             });
 
             if (!response.ok) {
-                throw Object.assign(new Error(response.statusText + ' => ' + (await response.text())));
+                const responseText = await response.text();
+                let errorMessage = response.status + ': ' + response.statusText;
+                errorMessage += responseText ? ' => ' + responseText : '';
```

Comment:
Had hard time to read the string formatting logic here. Could we use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)?